democracy_club/templates/about.html

```diff
@@ -13,97 +13,103 @@ <h1>About</h1>
 
-![Joe and Sym, Democracy Club coordinators](https://s14.postimg.org/n8t6twmk1/Joe_Sym_with_names_800x500.jpg "Sym and Joe")
+The club is supported by a core team of three coordinators:
 
-#### [Sym Roe](https://twitter.com/symroe)
-Sym is a developer and has worked in the open/government space for years. He runs Democracy Club partly by writing code and partly by working with others to build a community of organisations who want to work together.
+![From left: Sym, Joe and Chris — Democracy Club coordinators](https://democracyclub.org.uk/media/hermes/heroes/now-we-are-three_hero.jpg "Sym, Joe and Chris")
 
-#### [Joe Mitchell](https://twitter.com/j0e_m)
-Joe thinks that our most tricky political problems are systemically linked by weak democratic engagement. He thinks digital tools can begin to empower everyone — even those who don’t use them directly — to make democracy better. He has a background in public engagement, advocacy and public sector marketing.
 
-There are also ten to twenty core individuals who volunteer to make Democracy Club happen, two of whom are highlighted here:
+**Sym Roe ([@symroe](https://twitter.com/symroe))**
 
-#### [Tim Green](http://tfgg.me/)
-Tim worked on Democracy Club over the 2010 election and has created two other candidate sites, YourNextPCC and YourNextMEP. He thinks that lowering barriers to entry for understanding our system is critical if we want to improve UK democracy, and non-partisan online tools are the best way to achieve this.
+Sym has 15 years experience in open data, government and technology, for folks like Government Digital Service, Ministry of Justice and FarmSubsidy.org. He has never knowingly set foot in a school. He lives in Stroud and has still not been a candidate on the Great British Bake Off.
 
-#### [Mark Longair](https://twitter.com/mhl20)
-Mark is a developer working for mySociety on their parliaments team. He's been working on development of the code that powers Democracy Club’s candidates crowdsourcer, both as part of his day job and as a volunteer for Democracy Club.
+**Joe Mitchell ([@j0e_m](https://twitter.com/j0e_m))**
 
+Joe has a decade of experience in advocacy, communications and marketing, including with the UK Civil Service and United Nations. He went to many universities. He lives in Bristol, even though he is probably not cool enough to do so.
+
+**Chris Shaw**
+
+Chris has over 10 years of experience in software development and data wrangling in the research and education sectors. He studied Computer Science in Swansea, where he now lives. He started working on Democracy Club projects as a volunteer. We haven't worked out how to make him stop yet.
+
+The core team is overseen by an experienced board of directors:
+
+- Alice Casey
+- Alison Walters
+- Ian Watt
+- Mevan Babakar
+- Olly Benson
+- Rebecca Eligon
+- Rebecca Kemp
+- Sarah Hartley
+
+[Read about the board here.](https://democracyclub.org.uk/blog/2017/12/01/weve-got-board/)
+
+# Who funds us
```

Comment:
should this have a question mark?

Reply:
I think no, because it's in keeping with the other headers (like Who we are) ... though I agree it looks a bit odd...  maybe it should be Who us funds

Reply:
If you wanted a more naturally punctuation-less heading, "Our funders"?

Also, from a consistency perspective (if you're avoiding punctuation in header class elements) there are 2 level-3 headings with colons on the end:

* `### People who work on democracy trust us:`
* `### Here’s a roundup of articles about us:`